spec/locales-currencies-tz.html

```diff
@@ -9,11 +9,11 @@ <h1>Identification of Locales, Currencies, and Time Zones</h1>
     <h1>Case Sensitivity and Case Mapping</h1>
 
     <p>
-      The String values used to identify locales, currencies, and time zones are interpreted in a case-insensitive manner, treating the Unicode Basic Latin characters `"A"` to `"Z"` (U+0041 to U+005A) as equivalent to the corresponding Basic Latin characters `"a"` to `"z"` (U+0061 to U+007A). No other case folding equivalences are applied. When mapping to upper case, a mapping shall be used that maps characters in the range `"a"` to `"z"` (U+0061 to U+007A) to the corresponding characters in the range `"A"` to `"Z"` (U+0041 to U+005A) and maps no other characters to the latter range.
+      The String values used to identify locales, currencies, and time zones are interpreted in a case-insensitive manner, treating the Unicode Basic Latin characters *"A"* to *"Z"* (U+0041 to U+005A) as equivalent to the corresponding Basic Latin characters *"a"* to *"z"* (U+0061 to U+007A). No other case folding equivalences are applied. When mapping to upper case, a mapping shall be used that maps characters in the range *"a"* to *"z"* (U+0061 to U+007A) to the corresponding characters in the range *"A"* to *"Z"* (U+0041 to U+005A) and maps no other characters to the latter range.
     </p>
 
     <p>
-      EXAMPLES `"ß"` (U+00DF) must not match or be mapped to `"SS"` (U+0053, U+0053). `"ı"` (U+0131) must not match or be mapped to `"I"` (U+0049).
+      EXAMPLES *"ß"* (U+00DF) must not match or be mapped to *"SS"* (U+0053, U+0053). *"ı"* (U+0131) must not match or be mapped to *"I"* (U+0049).
```

Comment:
The only concern I see is that we may wish to change cases like <code>\*"ß"\* (U+00DF)</code> to <code>\`ß\` (U+00DF)</code> throughout this file, in alignment with, say, Note 4 of [21.2.2.8.2](https://tc39.es/ecma262/#sec-runtime-semantics-canonicalize-ch) in 262.

I don't have a strong opinion on this though, particularly since we're referring to these as "characters". (@jmdyck, what do you think?)

Reply:
It seems to me that it's incorrect for the preceding para to talk about ES String values as if they consist of Unicode code points.

Reply:
I believe this can be done through a separate patch if necessary.